railties/lib/rails/generators/rails/app/templates/config/initializers/content_security_policy.rb.tt

```diff
@@ -1,8 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
 # Define an application-wide content security policy.
-# For further information see the following documentation:
-# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+# See the Securing Rails Applications Guide for more information:
+# https://guides.rubyonrails.org/security.html
```

Comment:
Should this link to the CSP section?

```suggestion
# See the Content Security Policy section of the Rails Security Guide for more information:
# https://guides.rubyonrails.org/security.html#content-security-policy
```

Reply:
Oh, right, because we changed it to https://edgeguides.rubyonrails.org/security.html#content-security-policy-header in #44470.  Good catch!  Hopefully, it will be relatively stable going forward.